templates/config/arke/strategy.yaml

```diff
@@ -0,0 +1,15 @@
+strategy:
```

Comment:
If you put the whole file into app.yml, you can write one-liner here, just render @config['arke'].to_yaml

Reply:
I'm not sure we need to put the whole file into app.yml

Reply:
template is  wrong anyway, sources is a list, just keep it simple

Reply:
does it work? You may need to use gsub to delete `---` , also better delete rendered file :)